app/controllers/admin/articles_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Admin::ArticlesController < Admin::ApplicationController
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+
+    if @article.save(article_params)
+      flash[:notice] = "You successfully created a post."
+      redirect_to @article
+    else
+      flash[:notice] = "Error, please try again."
+      render :new
+    end
+  end
+
+  def show
```

Comment:
I would remove this action and instead have the admin panel link to the articles controller show instead